test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json

```diff
@@ -148,46 +148,46 @@
             }
         ],
         "timeRestore": false,
-        "title": "[Logs Apache] Access and error logs ECS",
```

Comment:
Strictly speaking, is this change needed as a result of updating the `package-spec` dependency? IIRC we don't yet have validation for this in the package spec, right?

Reply:
I admit I blindly copied over files from Integrations, which have been adjusted. If you prefer I can prepare anothe PR for spec to prevent `-ecs` suffixes and then bump up dependencies. WDYT?

Reply:
Yeah, I think I would prefer a separate PR. We're going to need it anyway to bump up dependencies when we implement validation for the absence of the `-ecs` prefix in the package spec (https://github.com/elastic/package-spec/issues/71). It would be nice if that PR also contained the corresponding changes to the test packages in `elastic-package` as well. That way, it's clearer what the change is about.

Reply:
I opened another PR with no-ECS-suffix validation: https://github.com/elastic/package-spec/pull/74

Reply:
The spec has been updated. Let me work on this PR to make it back to green.

Reply:
The PR is back to green and looks fine.